blocksuite/framework/store/src/model/workspace.ts

```diff
@@ -15,6 +16,8 @@ export interface Workspace {
   readonly idGenerator: IdGenerator;
   readonly blobSync: BlobEngine;
   readonly awarenessStore: AwarenessStore;
+  readonly loadDoc?: (doc: Y.Doc) => void;
```

Comment:
the names for `loadXXX` is not consistent with their signatures `(xx: Type) => void`, better to be `onLoadXXX`?